MAVProxy/modules/mavproxy_chat/assistant_setup/assistant_file_checker.py

```diff
@@ -64,20 +68,20 @@ def main(openai_api_key=None, delete_unused=False):
         for assistant_file in client.beta.assistants.files.list(assistant_id=assistant.id):
             file_assistant_count[assistant_file.id] = file_assistant_count.get(assistant_file.id, 0) + 1
             filename_size_date = get_filename_size_date_from_id(existing_files, assistant_file.id)

```

Comment:
You can pull the result apart directly here instead of indexing `filename_size_date`:
```suggestion
            (file_name, file_size, file_date) = get_filename_size_date_from_id(existing_files, assistant_file.id)
```